### PR TITLE
doc: update comment

### PR DESCRIPTION
**Pull request by deepthidevaki**
Part of https://github.com/zeebe-io/zeebe-chaos/pull/114#discussion_r810691392

### CODE REVIEW

File: chaos-workers/chaos-experiments/scripts/verify-readiness.sh [hunk 1]
```diff
@@ -8,13 +8,17 @@ namespace=$(getNamespace)
...
+  # Wait until brokers are ready. The component should be broker, but it is wrongly labelled as gateway
   kubectl wait --for=condition=Ready pod -l app.kubernetes.io/component=gateway --timeout=900s -n "$namespace"
-  # Wait until all gateway pods are running.
+  # Wait until all gateway pods are running. We cannot use `--for=condition=Ready` pod for gateway because
+  # this check also includes the evicted pods as long as they are not deleted. Evicted pods are not immediately
+  # deleted by kuberenetes. The following check passes only if all pods are running, and excludes evicted pods.
   kubectl wait --for=condition=Available deployment zeebe-gateway --timeout=120s -n "$namespace"
 else
-  # Wait until brokers are ready. The componenet should be broker, but it is wrongly labelled as gateway
+  # Wait until brokers are ready. The component should be broker, but it is wrongly labelled as gateway
```
Comment (oleschoenburg) [inline chaos-workers/chaos-experiments/scripts/verify-readiness.sh:18]: :upside_down_face: Here the component actually *is* broker so the comment is wrong.

**Review by oleschoenburg — APPROVED** [review]
Thanks @deepthidevaki, only a minor nitpick. Feel free to merge.

File: chaos-workers/chaos-experiments/scripts/verify-readiness.sh (hunk 1 above)
Comment (deepthidevaki) [reply chaos-workers/chaos-experiments/scripts/verify-readiness.sh:18]: In the command, we use the label `app.kubernetes.io/component=gateway` to identify brokers.

Comment (deepthidevaki) [reply chaos-workers/chaos-experiments/scripts/verify-readiness.sh:18]: Sorry. Got it :) It is for the non-cloud deployments :D

**Pull request merged**